openslides/core/static/js/core/core.js

```diff
@@ -605,6 +605,46 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.countdowns = [];
+    $scope.messages = [];
+    _.forEach(Projector.get(1).elements, function(element) {
+        if (element.name == 'core/countdown') {
+            $scope.countdowns.push(element);
+        }
+        if (element.name == 'core/message') {
+            $scope.messages.push(element);
+        }
+    });
+    $scope.addCountdown = function () {
+        $http.post(
+            '/rest/core/projector/1/activate_elements/',
+            [{name: 'core/countdown', status: 'stop', visible: true,
+                countdown_time: 42, stable: false}]
+            // TODO countdown_time: config auslesen
```

Comment:
You take it from OpenSlides config, not from projector model. See openslides.core.signals.py